librosa/effects.py

```diff
@@ -374,3 +377,143 @@ def remix(y, intervals, align_zeros=True):
         y_out.append(y[clip])
 
     return np.concatenate(y_out, axis=-1)
+
+
+def trim(y, top_db=60, ref_power=np.max, n_fft=2048, hop_length=512,
+         index=False):
+    '''Trim leading and trailing silence from an audio signal.
+
+    Parameters
+    ----------
+    y : np.ndarray, shape=(n,) or (2,n)
+        Audio signal, can be mono or stereo
+
+    top_db : number > 0
+        The threshold (in decibels) below reference to consider as
+        silence
+
+    ref_power : number or callable
+        The reference power.  By default, it uses `np.max` and compares
+        to the peak power in the signal.
+
+    n_fft : int > 0
```

Comment:
Agreed; `frame_length` would match `util.frame`, but it propagates `n_fft` because that's how it's called in `rmse`.

Of course, we could (should) rename the parameter in `rmse` as well, since it's no longer accurate.  I'll handle that in a separate PR.